src/MidiPlayer.js

```diff
@@ -624,4 +625,36 @@ not.
         this.logging = logging;
         this.eventHandler.setLogger({ eventLogger, logging });
     }
+    /*
+     * get total duration time
+     * 2021 03 NormalUniversity
+     */
+    getDuration() {
+
+        return this.totalTime;
+    }
+    /**
+     * Seek Position.
+     * @param {any} [mSecond] milli seconds to be seek
+     * @example
+     *  let midiPlayer = new MidiPlayer({ logging: false, patchUrl: 'lib/pat/' });
+     *  midiPlayer.play({url: $url})
+     *  midiPlayer.seek({mSecond: $milliSeconds});
+     */
+    async seek({ mSecond}) {
+        if (this.totalTime>0) {
+            let _mSecond = this.sampleRate * mSecond;
```

Comment:
The use of the `_` prefix to signify private variables is not used anywhere else in the library. Please stick to existing conventions. As noted before, the word "millisecond" is a single word so no "S" is needed. I would suggest using `msecond` as a name for this variable.